xbmc/games/addons/cheevos/GameClientCheevos.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+ *  Copyright (C) 2020-2021 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "GameClientCheevos.h"
+
+#include "addons/kodi-dev-kit/include/kodi/c-api/addon-instance/game.h"
+#include "games/addons/GameClient.h"
+
+using namespace KODI;
+using namespace GAME;
+
+constexpr int HASH_SIZE = 33;
+constexpr int URL_SIZE = 512;
+constexpr int RICH_PRESENCE_EVAL_SIZE = 512;
+constexpr int POST_DATA_SIZE = 1024;
```

Comment:
By use of `char**` and `strdup`/`free` are the `constexpr int` values no more needed.